packages/beacon-node/src/chain/validation/blobSidecar.ts

```diff
@@ -211,12 +225,32 @@ export async function validateBlockBlobSidecars(
     );
   }
 
+  if (chain !== null) {
+    const headState = await chain.getHeadState();
+    const signatureSet = getBlockHeaderProposerSignatureSet(headState, firstSidecarSignedBlockHeader, true);
+
+    if (
+      !(await chain.bls.verifySignatureSets([signatureSet], {
+        batchable: true,
+        priority: false,
```

Comment:
```suggestion
        priority: true,
```
because in `downloadByRoot` is more important than verifying other gossip messages
while in `downloadByRange` it does not matter much